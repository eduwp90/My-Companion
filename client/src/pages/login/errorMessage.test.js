import { render } from '@testing-library/react';
import ErrorMessage from './errorMessage';

describe ('ErrorMessage', () => {
  test ('ErrorMessage should match snapshot', () => {
    const { container } = render(<ErrorMessage />);
    expect (container.firstChild).toMatchSnapshot();
  })
})