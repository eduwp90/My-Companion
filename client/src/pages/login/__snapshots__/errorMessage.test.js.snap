// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage ErrorMessage should match snapshot 1`] = `
<div
  class="css-pb69ky"
>
  <div
    class="chakra-alert css-el9lkb"
    role="alert"
  >
    <span
      class="chakra-alert__icon css-1c6vscd"
    >
      <svg
        class="chakra-icon css-fi0ww0"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M11.983,0a12.206,12.206,0,0,0-8.51,3.653A11.8,11.8,0,0,0,0,12.207,11.779,11.779,0,0,0,11.8,24h.214A12.111,12.111,0,0,0,24,11.791h0A11.766,11.766,0,0,0,11.983,0ZM10.5,16.542a1.476,1.476,0,0,1,1.449-1.53h.027a1.527,1.527,0,0,1,1.523,1.47,1.475,1.475,0,0,1-1.449,1.53h-.027A1.529,1.529,0,0,1,10.5,16.542ZM11,12.5v-6a1,1,0,0,1,2,0v6a1,1,0,1,1-2,0Z"
          fill="currentColor"
        />
      </svg>
    </span>
    <div
      class="chakra-alert__desc css-6n7j50"
    />
  </div>
</div>
`;
